demo-extended/src/main/java/com/novoda/downloadmanager/demo/extended/batches/Batch.java

```diff
@@ -0,0 +1,51 @@
+package com.novoda.downloadmanager.demo.extended.batches;
+
+import com.novoda.downloadmanager.lib.DownloadManager;
+
+public class Batch {
```

Comment:
just add javadoc to the classes with your comments about the difference Andrei , javadoc in demo;s is cool